<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsClass</class>
 <widget class="QDialog" name="settingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>337</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_UDPNotifications">
     <property name="title">
      <string>UDP notifications</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_ImageQuality">
        <property name="text">
         <string>image quality</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="inputMask">
         <string>00000; </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="l_1">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_DriftData">
        <property name="text">
         <string>drift data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_GuidingStability">
        <property name="text">
         <string>guiding stability</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_StartStop">
        <property name="text">
         <string>start/stop</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="l_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="inputMask">
         <string>000.000.000.000;_</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_Server">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Command Server</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="lineEdit_lsocket">
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_useTCP">
        <property name="text">
         <string>use  TCP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="l_lsocket">
        <property name="text">
         <string>Unix Socket</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="l_lport">
        <property name="text">
         <string>Listen Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="lineEdit_TCPPort">
        <property name="inputMask">
         <string>00000; </string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_Misc">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="3">
       <widget class="QCheckBox" name="checkBox_DBGVerbosity">
        <property name="text">
         <string>debug verbosity</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="comboBox_DriftGraph_nx"/>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="spinBox_DitherRange">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dithering timeout, sec</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Dithering range, pix</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBox_DitherRestTout">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Drift graph X cells</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Drift graph Y cells</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboBox_DriftGraph_ny"/>
      </item>
      <item row="6" column="3">
       <widget class="QCheckBox" name="checkBox_ShowHelperTB">
        <property name="text">
         <string>show Helper toolbar</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QCheckBox" name="checkBox_HFD_on">
        <property name="text">
         <string>calculate HFD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_IP</tabstop>
  <tabstop>lineEdit_Port</tabstop>
  <tabstop>checkBox_StartStop</tabstop>
  <tabstop>checkBox_GuidingStability</tabstop>
  <tabstop>checkBox_ImageQuality</tabstop>
  <tabstop>checkBox_DriftData</tabstop>
  <tabstop>lineEdit_lsocket</tabstop>
  <tabstop>checkBox_useTCP</tabstop>
  <tabstop>lineEdit_TCPPort</tabstop>
  <tabstop>spinBox_DitherRange</tabstop>
  <tabstop>spinBox_DitherRestTout</tabstop>
  <tabstop>comboBox_DriftGraph_nx</tabstop>
  <tabstop>comboBox_DriftGraph_ny</tabstop>
  <tabstop>checkBox_HFD_on</tabstop>
  <tabstop>checkBox_ShowHelperTB</tabstop>
  <tabstop>checkBox_DBGVerbosity</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_useTCP</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_TCPPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_useTCP</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_lsocket</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
